modules/javafx.controls/src/main/java/javafx/scene/control/skin/TabPaneSkin.java

```diff
@@ -2289,4 +2289,9 @@ private void stopAnim(Animation anim) {
     ContextMenu test_getTabsMenu() {
         return tabHeaderArea.controlButtons.popup;
     }
+
+    void test_disableAnimations() {
```

Comment:
Might be better to change this to `test_setAnimation(TabAnimation mode)` so the test can restore it in the cleanup method.

Reply:
Never mind. I see that the test constructs a new TabPane instance for each test, so there is no need to restore it.